build/Projects.cs

```diff
@@ -13,6 +13,12 @@ public static class Tests
         public const string AutoInstrumentationLoaderTests = "OpenTelemetry.AutoInstrumentation.Loader.Tests";
         public const string AutoInstrumentationBootstrappingTests = "OpenTelemetry.AutoInstrumentation.Bootstrapping.Tests";
         public const string AutoInstrumentationTests = "OpenTelemetry.AutoInstrumentation.Tests";
+        public const string IntegrationTests = "IntegrationTests";
+
+        public static class Applications
```

Comment:
Hum, in light of the other comment above a name that makes clear that this is being singled out because of the usage of `packages.config` seems better. It also should be a collection. This way we can add any other managed project that uses one here and things should just work - instead of adding the project to the concat where this one is added.

Reply:
It's actually singled out for 2 purposes: 1 is the `packages.config` the second is for `Build.Steps.Windows`.
Why I didn't make all helpers `IEnumerable<Project>` is that, I wasn't sure if there will be additions in the near future. When it happens there could be need for some additional refactor to introduce a pattern etc. So I applied YAGNI at this moment.

Reply:
I guess `packages.config` would only ever apply on Windows, but, YAGNI may be good for now :)